src/index.js

```diff
@@ -23,6 +25,8 @@ export { default as FieldLabel } from './components/field-label';
 
 export { default as TextField } from './components/fields/text-field';
 
+export { Icons };
```

Comment:
This is the new `Icons` export

Reply:
I wonder if it would be worth following up here with possibly code splitting this?